src/types.ts

```diff
@@ -545,16 +545,39 @@ export enum TigrisDataTypes {
 	OBJECT = "object",
 }
 
-export interface TigrisFieldOptions {
-	maxLength?: number;
+export enum FieldDefaults {
+	TIME_UPDATED_AT = "updatedAt",
```

Comment:
Do we need `TIME_` prefix or can we just have `UPDATED_AT`, `CREATED_AT`, and `NOW`